packages/react-native-reanimated/android/src/main/java/com/swmansion/worklets/WorkletsModule.java

```diff
@@ -69,6 +69,12 @@ public boolean installTurboModule(String valueUnpackerCode) {
   }
 
   public void invalidate() {
+    // We have to destroy extra runtimes when invalidate is called. If we clean
+    // it up later instead there's a chance the runtime will retain references
+    // to invalidated memory and will crash on its destruction.
+    invalidateCpp();
     mAndroidUIScheduler.deactivate();
```

Comment:
Let's add a newline here as well
```suggestion
    // We have to destroy extra runtimes when invalidate is called. If we clean
    // it up later instead there's a chance the runtime will retain references
    // to invalidated memory and will crash on its destruction.
    invalidateCpp();

    mAndroidUIScheduler.deactivate();
```